.cirrus.yml

```diff
@@ -270,8 +282,15 @@ qa_task:
     ffmpeg -loglevel warning -f x11grab -video_size 1280x960 -i ${DISPLAY} -codec:v libx264 -r 12 ${CIRRUS_WORKING_DIR}/recording_${IDEA_VERSION}.mp4
   run_its_script: |
     echo "Run ITs on ${IDEA_VERSION}"
+    echo "Run ITs on ${QA_CATEGORY}"
     source .cirrus/use-gradle-wrapper.sh
-    gradle :its:check --stacktrace -i -PijVersion=${IDEA_VERSION} -PslPluginDirectory=${CIRRUS_WORKING_DIR}/staged-plugin
+    if [[ ${QA_CATEGORY} = "Idea2022_Standalone" ]] || [[ ${QA_CATEGORY} = "Idea2023_Standalone" ]]; then
+    gradle its:test --tests StandaloneIdeaTests --stacktrace -i -PijVersion=${IDEA_VERSION} -PslPluginDirectory=${CIRRUS_WORKING_DIR}/staged-plugin
```

Comment:
What's the reason to use `test` instead of `check`?

Reply:
Check only runs tests by default; if there are other verifications, it also runs them. In our case, we do have other verifications like licenseFormat for ITs. So I am running the check without the tests as a separate task. Since we are running tests parallelly it is better to run and also add new parallel tests by using the test instead of check since we are running them explicitly. Excluding tests from check would become verbose for parallel execution.